contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -613,15 +608,38 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
     return (weiPerUnitLink, isFeedStale);
   }
 
+  /**
+   * @notice Returns the block number of the current block by using specific opcode.
+   * @notice Override this function in chain specific way if needed (L2 chains).
+   */
+  function _getBlockNumber() internal view virtual returns (uint256) {
+    return block.number;
+  }
+
+  /**
+   * @notice Returns the blockhash for the given blockNumber by using specific opcode.
+   * @notice If the blockNumber is more than 256 blocks in the past, returns the empty string.
+   * @notice Override this function in chain specific way if needed (L2 chains).
+   */
+  function _getBlockhash(uint64 blockNumber) internal view virtual returns (bytes32) {
+    return blockhash(blockNumber);
+  }
+
+  /**
+   * @notice Returns the L1 fee for the calldata payload (always return 0 on L1 chains).
+   * @notice Override this function in chain specific way for L2 chains.
+   */
+  // solhint-disable-next-line no-unused-vars
```

Comment:
Do we still need this solhint disable if we commented out the variable?

Reply:
Nope, looks like I can remove it and the compiler no longer complains. Thanks!